addons/crm_iap_mine/static/src/views/generate_leads_hook.js

```diff
@@ -1,27 +0,0 @@
-/** @odoo-module **/
-
-import { useService } from "@web/core/utils/hooks";
-
-const { onWillStart, useComponent } = owl;
-
-export function useGenerateLeadsButton() {
```

Comment:
where was this function used?

Reply:
in crm_iap_mine/static/src/views/*.js. But we remove it.